OpenRA.Mods.Common/Widgets/Logic/IntroductionPromptLogic.cs

```diff
@@ -25,11 +26,13 @@ public class IntroductionPromptLogic : ChromeLogic
 		[FluentReference]
 		const string Classic = "options-control-scheme.classic";
 
+		[FluentReference]
+		const string Generals = "options-control-scheme.generals";
+
 		[FluentReference]
 		const string Modern = "options-control-scheme.modern";
 
-		readonly string classic;
-		readonly string modern;
+		readonly Dictionary<MouseControlType, string> controlTypes;
```

Comment:
this could just be an internal variable that's captured by the lambda

Reply:
Which lambda? The control types are set in a different scope than the one they are used in, and the key strings that FluentProvider.GetMessage() uses aren't just the enum name string, so unless you want be to use reflection to get to those key strings I'm not sure a better way to handle this